src/components/search-box/SearchBox.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import './SearchBox.scss';
+
+import Search from '../search';
+
+const SearchBox = () => {
+    return (
+        <div className="search-box-container">
```

Comment:
Shouldn't this be a `<form>`?

Reply:
Fixed and added `aria-label` to conform with prod.